ui/retribution_paladin/sim.ts

```diff
@@ -128,24 +128,27 @@ const SPEC_CONFIG = registerSpecConfig(Spec.SpecRetributionPaladin, {
 
 	presets: {
 		rotations: [
+			...Presets.APLPresets[Phase.Phase5],
 			...Presets.APLPresets[Phase.Phase4],
-			...Presets.APLPresets[Phase.Phase3],
-			...Presets.APLPresets[Phase.Phase2],
-			...Presets.APLPresets[Phase.Phase1],
+			//...Presets.APLPresets[Phase.Phase3],
+			//...Presets.APLPresets[Phase.Phase2],
+			//...Presets.APLPresets[Phase.Phase1],
```

Comment:
We should still leave all of the old presets available for potentially lower level simming

Reply:
Since there's SO many paladin presets especially in phase 4, what we can do is apply level restrictions to them. Take a look at https://github.com/wowsims/sod/blob/master/ui/mage/presets.ts if you'd like an example of that. It'll show only relevant presets based on the player's selected level

Reply:
I did not know that was available, it is much better this way